### PR TITLE
Second version of switching things to cloud build.

### PR DESCRIPTION
**Pull request by dlorenc**
Ref #29 #35 
cc @duggelz 

This version is a lot simpler but it builds the interpreter each time and takes about 35 minutes to run.

### CODE REVIEW

**Comment by jonparrott** [comment]
+1, I like this as a good first step.

The next thing we can do is have the interpreter builder upload to GCS, generate a signature, and use that to determine if it needs to rebuilt.

**Comment by duggelz** [comment]
+1

**Comment by duggelz** [comment]
Is this a real error or just a symptom of a timeout?

Fetched 165 MB in 35s (4593 kB/s)
[91mE: Failed to fetch http://httpredir.debian.org/debian/pool/main/b/blas/libblas-common_1.2.20110419-10_amd64.deb  Error reading from server. Remote end closed connection [IP: 5.153.231.35 80]

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
[0mThe command '/bin/sh -c /scripts/install-apt-packages.sh' returned a non-zero code: 123
make: *** [build] Error 123

**Comment by dlorenc** [comment]
> Fetched 165 MB in 35s (4593 kB/s)
> �[91mE: Failed to fetch http://httpredir.debian.org/debian/pool/main/b/blas/libblas-common_1.2.20110419-10_amd64.deb Error reading from server. Remote end closed connection [IP: 5.153.231.35 80]
> 
> E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
> �[0mThe command '/bin/sh -c /scripts/install-apt-packages.sh' returned a non-zero code: 123
> make: *** [build] Error 123

That looks like some kind of timeout. I saw that (although a different package) yesterday at least once. The httpredir server seems to be less stable than our old gce_debian_mirror, but that is getting turned down at some point.

**Comment by jonparrott** [comment]
I would generally just ignore travis for now. Manually verify or use jenkins.

**Comment by jonparrott** [comment]
I turned travis off.

**Comment by dlorenc** [comment]
Should I merge this?

**Comment by duggelz** [comment]
Let me finish my release first, then yes.

File: Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ ENV PYTHONUNBUFFERED 1
 RUN pip install --upgrade pip virtualenv
 
 # Install the Google-built interpreters
-ADD python-interpreter-builder/output/interpreters.tar.gz /
+ADD interpreters.tar.gz /
```
Comment (duggelz) [inline Dockerfile]: Doesn't this break the (non-cloud) build target?

Comment (duggelz) [inline Dockerfile]: No, I was wrong, I didn't see the change below

**Comment by duggelz** [comment]
Merging now...

**Pull request merged**